sources/assets/shells/aliases.d/fzf

```diff
@@ -1,4 +1,5 @@
 alias fzf-wordlists='find /opt/rockyou.txt /opt/seclists /usr/share/wordlists /usr/share/wfuzz /usr/share/dirb -type f | fzf'
```

Comment:
`/opt/seclists` can be removed here since seclists will be in `/opt/lists` (@QU35T-code was right, it's a good idea)

Reply:
Here i remove  `/opt/rockyou.txt` because now `rockyou.txt` is in  `/opt/lists/rockyou.txt` and since i already added `/opt/lists`  `/opt/rockyou.txt` is no longer relevant